src/Examples.Console/Examples.Console.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
```

```suggestion
    <IsPackable>false</IsPackable>
    <OutputType>Exe</OutputType>
```

Then we could simply do `dotnet pack` without passing the path to the `.csproj` since that's the only packable project in the solution.

---

that makes a lot of sense, `dotnet expert` on the case 😄